core/src/main/java/org/apache/calcite/sql/type/ReturnTypes.java

```diff
@@ -615,10 +615,11 @@ public static SqlCall stripSeparator(SqlCall call) {
       ARG0.andThen(SqlTypeTransforms.TO_MULTISET);
 
   /**
-   * Returns the element type of a MULTISET.
+   * Returns the element type of a MULTISET, with nullability enforced.
    */
   public static final SqlReturnTypeInference MULTISET_ELEMENT_NULLABLE =
```

Comment:
Is the name appropriate?

There are two field:
- NULLABLE -> TO_NULLABLE
```
  public static final SqlReturnTypeInference TO_COLLECTION_ELEMENT_NULLABLE =
      TO_COLLECTION_ELEMENT.andThen(SqlTypeTransforms.TO_NULLABLE);
```
FORCE_NULLABLE -> FORCE_NULLABLE
```
  public static final SqlReturnTypeInference TO_COLLECTION_ELEMENT_FORCE_NULLABLE =
      TO_COLLECTION_ELEMENT.andThen(SqlTypeTransforms.FORCE_NULLABLE);
```

Reply:
Yes, we should use old name.

I like using a new field to fix it. 

Please squash commits and I will merge it soon.

Reply:
@JiajunBernoulli done rebasing/squashing. Thank you.